lib/ymchat_flutter.dart

```diff
@@ -200,8 +200,8 @@ class YmChat {
   }
 
   static Future<bool> setThemeBotBubbleBackgroundColor(String color) async {
-    bool isThemeBotBubbleBackgroundColorAssigned =
-        await _channel.invokeMethod('setThemeBotBubbleBackgroundColor', {"color": color});
+    bool isThemeBotBubbleBackgroundColorAssigned = await _channel
+        .invokeMethod('setThemeBotBubbleBackgroundColor', {"color": color});
```

Comment:
This change was not required in this

Reply:
It's get added because of prettier formatting